test/pkg.jl

```diff
@@ -19,14 +19,17 @@ import Pkg3.Types.semver_spec
 @testset "semver notation" begin
     @test semver_spec("^1.2.3") == Pkg3.Types.VersionSpec("1.2.3-2")
     @test semver_spec("^1.2")   == Pkg3.Types.VersionSpec("1.2.0-2")
-    @test semver_spec("1")      == Pkg3.Types.VersionSpec("1.0.0-2")
+    @test semver_spec("^1")     == Pkg3.Types.VersionSpec("1.0.0-2")
     @test semver_spec("^0.2.3") == Pkg3.Types.VersionSpec("0.2.3-0.3")
     @test semver_spec("0.0.3")  == Pkg3.Types.VersionSpec("0.0.3-0.0.4")
```

Comment:
and maybe here too, although in this case there's no difference between `^` and `=`.